lib/gitarro/backend.rb

```diff
@@ -178,6 +183,10 @@ def print_pr_resume(prs)
     end
   end
 
+  def print_test_required
+    puts '[TESTRQ=true] PR requires test'
```

Comment:
can we have TESTREQUIRED ? it is more clear

Reply:
Sure!